### PR TITLE
449 Redelegation Documentation

### PR DESCRIPTION
**Pull request by ACStoneCL**
### Related links

[Documentation explaining the redelegate feature incl. the relevant session code #449](https://github.com/casper-network/docs/issues/449)

### Changes

Documentation explaining the new `redelegate.wasm` and associated re-delegation process using the `Delegating with the Command-line` document as a template.

### Notes

Ran locally without issue.

### CODE REVIEW

File: source/docs/casper/workflow/redelegate.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+--node-address http://<peer-ip-address>:7777/rpc
+```
+
+**Request fields**:
+
+-   `node-address` - <HOST:PORT> Hostname or IP and port of node on which HTTP service is running \[default:<http://localhost:7777>\]
+
+The `get-auction-info` call will return all the bids currently in the auction contract and the list of active validators for `4` future eras from the present era.
+
+Below is a sample output:
```
Comment (ipopescu) [inline source/docs/casper/workflow/redelegate.md:111]: You could collapse this if you want.

File: source/docs/casper/workflow/redelegate.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+Once the deploy has been executed, we can query the auction for information to confirm our re-delegation. We can use the Casper command-line client to create an RPC request with the following query:
+
+```bash
+casper-client get-auction-info \
+--node-address http://<peer-ip-address>:7777/rpc
+```
+
+**Request fields**:
+
+-   `node-address` - <HOST:PORT> Hostname or IP and port of node on which HTTP service is running \[default:<http://localhost:7777>\]
```
Comment (ipopescu) [inline source/docs/casper/workflow/redelegate.md:107]:
```suggestion
-   `node-address` - <HOST:PORT> Hostname or IP and port of node on which HTTP service is running \[default: <http://localhost:7777>\]
```

File: source/docs/casper/workflow/redelegate.md (hunk 1 above)
Comment (ACStoneCL) [reply source/docs/casper/workflow/redelegate.md:111]: I think the snippets are short enough to keep un-collapsed - mostly just due to the following text referencing them.

Comment (ipopescu) [reply source/docs/casper/workflow/redelegate.md:111]: Ok!

**Review by darthsiroftardis — APPROVED** [review]
Content related to Redelegate looks good, but lets resolve the git stuff before we merge

**Pull request merged**